tests/python/unittest/test_random.py

```diff
@@ -61,6 +61,7 @@ def check_with_device(device, dtype):
         },
         {
             'name': 'randn',
+            'symbol': mx.sym.random.randn,
             'ndop': mx.nd.random.randn,
             'params': { 'loc': 10.0, 'scale': 0.5 },
```

Comment:
I think we should have test for the newly added feature here.

Reply:
Makes sense!